lib/ws_servers/api/index.js

```diff
@@ -82,7 +82,7 @@ module.exports = class APIWSServer extends WSServer {
     this.logAlgoOpts = logAlgoOpts
     this.config = config
 
-    this.reconnectAlgoHost = this._reconnectAlgoHost.bind(this)
+    this._reconnectAlgoHost = this._reconnectAlgoHost.bind(this)
```

Comment:
we are using `reconnectAlgoHost` in many places .. changes here will require the name to be changed in various places as well
https://github.com/bitfinexcom/bfx-hf-server/blob/master/lib/ws_servers/api/handlers/on_save_api_credentials.js
https://github.com/bitfinexcom/bfx-hf-server/blob/master/lib/ws_servers/api/handlers/on_settings_update.js

Reply:
I was fooled by my IDE saying that this was not used anymore, my bad

Reply:
0d85c4bf01e3bfb95f666ea7b08c9c542eeef384